### PR TITLE
Fixes broken links to API docs

### PR DESCRIPTION
**Pull request by almcd**
## Description
Fixes two broken links to API docs in the "Video.js Documentation" file.

## Specific Changes proposed
Updated the links to what I believe to be logical replacement URLs.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -33,5 +33,5 @@ There are two categories of docs: [Guides](./guides/) and [API docs](./api/). Gu
 * [Removing Players](./guides/removing-players.md) - Helpful for using VideoJS in single page apps.
 
 ## API Docs
-- The most relevant API doc is the [player API doc](./api/vjs.Player.md).
-- [Full list of API Docs](./api/)
+- The most relevant API doc is the [player API doc](./guides/api.md).
```
Comment (gkatsev) [inline docs/index.md]: This makes the title and the link not match. The most relevant API doc *is* the player api doc as well.

**Comment by almcd** [comment]
@gkatsev Following your feedback, I've removed the other link. Hopefully this makes more sense.

**Comment by misteroneill** [comment]
Thanks! LGTM

**Comment by BrandonOCasey** [comment]
LGTM

**Comment by gkatsev** [comment]
I don't think removing that line is correct either. Most videojs users only care about the player API docs.

File: docs/index.md [hunk 2]
```diff
@@ -33,5 +33,4 @@ There are two categories of docs: [Guides](./guides/) and [API docs](./api/). Gu
 * [Removing Players](./guides/removing-players.md) - Helpful for using VideoJS in single page apps.
 
 ## API Docs
-- The most relevant API doc is the [player API doc](./api/vjs.Player.md).
```
Comment (gkatsev) [inline docs/index.md]: we should just link to the player doc: http://docs.videojs.com/docs/api/player.html

**Comment by gkatsev** [comment]
Merged via https://github.com/videojs/video.js/commit/4063f96bb5581123959fd17c10f666bf2bf17ebd

**Pull request closed without merging**